services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/HankeControllerTest.kt

```diff
@@ -61,22 +60,22 @@ class HankeControllerTest {
     }
 
     @Test
-    fun `test that the getAllHankeItems returns ok and two items`() {
-
-        var listOfHanke =  listOf(
-            Hanke(1234, mockedHankeTunnus, true,
-                    "Mannerheimintien remontti remonttinen", "Lorem ipsum dolor sit amet...",
-                    getDatetimeAlku(), getDatetimeLoppu(), Vaihe.OHJELMOINTI, null,
-                    1, "Risto", getCurrentTimeUTC(), null, null, SaveType.DRAFT),
-            Hanke(50, "HAME50", true,
-                    "Hämeenlinnanväylän uudistus", "Lorem ipsum dolor sit amet...",
-                    getDatetimeAlku(), getDatetimeLoppu(), Vaihe.SUUNNITTELU, SuunnitteluVaihe.KATUSUUNNITTELU_TAI_ALUEVARAUS,
-                    1, "Paavo", getCurrentTimeUTC(), null, null, SaveType.SUBMIT)
+    fun `test when called without parameters then getAllHankeItems returns ok and two items`() {
+
+        var listOfHanke = listOf(
+                Hanke(1234, mockedHankeTunnus, true,
+                        "Mannerheimintien remontti remonttinen", "Lorem ipsum dolor sit amet...",
+                        getDatetimeAlku(), getDatetimeLoppu(), Vaihe.OHJELMOINTI, null,
+                        1, "Risto", getCurrentTimeUTC(), null, null, SaveType.DRAFT),
+                Hanke(50, "HAME50", true,
+                        "Hämeenlinnanväylän uudistus", "Lorem ipsum dolor sit amet...",
+                        getDatetimeAlku(), getDatetimeLoppu(), Vaihe.SUUNNITTELU, SuunnitteluVaihe.KATUSUUNNITTELU_TAI_ALUEVARAUS,
+                        1, "Paavo", getCurrentTimeUTC(), null, null, SaveType.SUBMIT)
         )
 
         Mockito.`when`(hankeService.loadAllHanke()).thenReturn(listOfHanke)
 
-        val response: ResponseEntity<Any> = hankeController.getAllHankeItems()
+        val response: ResponseEntity<Any> = hankeController.getHankeList(null)
```

Comment:
Could the parameter be null as default? So this would be hankeController.getHankeList()